config/rbac/service-account.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Would you mind generating this new service account stuff in a separate PR, just to clean this one up a little bit?

Reply:
Regenerated controller in https://github.com/aws-controllers-k8s/ecr-controller/pull/32